plugin/Controller/EstateDetailUrl.php

```diff
@@ -35,6 +35,7 @@ class EstateDetailUrl
 	 * @param string|null $title
 	 * @param string|null $oldUrl
 	 * @param bool $flag
+	 * @param string|null $switchLocale
```

Comment:
Is type null needed?

Reply:
@andernath It really isn't necessary. I'll fix it.